### DISCUSSION TITLE
How to use $today function to find matching JSON value?

### DISCUSSION THREAD

**Discussion by ryzen3100** [mavoweb/mavo#934] [Q&A] [answered]
Hi,
How to use the $today function to parse a matching JSON value? (the $today would return "day X")
Example:
```
<pre id="data" hidden>{
        "2022-10-16": "day A",
        "2022-10-17": "day B",
        "2022-10-18": "day C",
        "2022-10-19": "day D",
        "2022-10-20": "day E"

    }</pre>

<div mv-app="dataStorage" mv-storage="#data">
  [set(tDay, $today)]
    <span property="tDay">#</span>

  <p>Data match for today: [tDay]</p>
 </div>
```

**Comment by LeaVerou** [comment, marked as answer]
Please note that `set()` will not work in an expression, it's an action function and will exclusively work in [Actions](https://mavo.io/docs/actions).

Your Mavo template does not seem to correspond to the data you have. The JSON this Mavo app would produce is 

```json
{
	"tDay": "#"
}
```

If your data comes from elsewhere, you can do `[get(dataStorage, $today)]` to get the day name, but do note Mavo cannot create such an object with variable property names (except perhaps with a bunch of workarounds).
